### PR TITLE
chore: Explorer migration to Unity 2019.4.0f1

### PR DESCRIPTION
**Pull request by sandrade-dcl**
# What? 
Migration of the current client to Unity 2019.4.0f1.

### CODE REVIEW

**Review by eordano — APPROVED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Components/Animator/Tests/AnimatorTests.cs [hunk 1]
```diff
@@ -12,6 +12,16 @@ namespace Tests
...
+        protected override IEnumerator SetUp()
+        {
+            yield return base.SetUp();
+        }
+
+        protected override IEnumerator TearDown()
+        {
+            yield return base.TearDown();
+        }
+
```
Comment (pravusjif) [inline unity-client/Assets/Scripts/MainScripts/DCL/Components/Animator/Tests/AnimatorTests.cs]: These look like unneeded overrides

File: unity-client/Assets/Scripts/MainScripts/DCL/Components/Materials/Tests/MaterialsTests.cs [hunk 2]
```diff
@@ -14,8 +14,13 @@ public class MaterialsTests : TestsBase
...
         {
-            yield return SetUp_SceneController(true, false);
-            yield return SetUp_CharacterController();
+            yield return base.SetUp();
+            SceneController.i.useBoundariesChecker = false;
+        }
+
+        protected override IEnumerator TearDown()
+        {
+            yield return base.TearDown();
```
Comment (pravusjif) [inline unity-client/Assets/Scripts/MainScripts/DCL/Components/Materials/Tests/MaterialsTests.cs]: this looks like an unneeded override

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Tests/AvatarEditorHUDControllerShould.cs [hunk 3]
```diff
@@ -147,8 +134,8 @@ public void LoadUserProfileByConstructor()
                 }
             }, false);
 
-            controller = new AvatarEditorHUDController_Mock();
-            controller.Initialize(userProfile, catalog);
+            // controller = new AvatarEditorHUDController_Mock();
+            // controller.Initialize(userProfile, catalog);
```
Comment (pravusjif) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Tests/AvatarEditorHUDControllerShould.cs]: let's remove this commented code

**Review by pravusjif — APPROVED** [review]

**Review by sandrade-dcl — COMMENTED** [review]
It looks very fine, good job Brian! :)

**Pull request merged**